### ISSUE TITLE
Add accept attribute to Form.FileInput

### ISSUE THREAD

**Issue by joaquinwojcik** [tabler/tabler-react#418]
Hi! Greetings from Argentina.

First, I want to thank all the colaborators of this project. Its just amazing.

Now, I'm trying to use the [accept attribute for "input file"](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#attr-accept) for only allow txt files, but I've noticed that Form.FileInput doesn't have support for this.

**Code:**
```
<Form.FileInput name='file' label='Seleccione un archivo' accept=".txt" />
```

**Result:**
```
<div class="custom-file">
   <input type="file" class="custom-file-input" name="file" value="">
   <label class="custom-file-label" style="white-space: nowrap; display: block; overflow: hidden;">Seleccione un archivo</label
</div>
```

**Expected result:** 
```
<div class="custom-file">
   <input type="file" class="custom-file-input" name="file" value="" accept=".txt">
   <label class="custom-file-label" style="white-space: nowrap; display: block; overflow: hidden;">Seleccione un archivo</label
</div>
```

I dont know what files should I modify. Can anyone guide me? 
Thanks!

**Comment by GregoryHamel** [comment]
@joaquinwojcik, that looks like a simple addition of props! Please make a PR with your changes once you are done.

The file you are looking for is `./src/components/Form/FormFileInput.react.js`

**Comment by joaquinwojcik** [comment]
Thank you @GregoryHamel ! 

Im getting this error when try to commit my changes.

```
Warning: Setting pre-commit script in package.json > scripts will be deprecated
Please move it to husky.hooks in package.json, a .huskyrc file, or a husky.config.js file
Or run ./node_modules/.bin/husky-upgrade for automatic update

See https://github.com/typicode/husky for usage

husky > pre-commit (node v8.10.0)
yarn run v1.13.0
$ pretty-quick --staged
🔍  Finding changed files since git revision 0928f51.
🎯  Found 1 changed file.
✅  Everything is awesome!
Done in 2.57s.
  ↓ Stashing changes... [skipped]
    → No partially staged files found...
  ✔ Running linters...
All files checked!
./scripts/runflow: line 2: $'\r': command not found
: ambiguous redirectine 3: 1
./scripts/runflow: line 5: $'\r': command not found
: ambiguous redirectine 6: 1
./scripts/runflow: line 8: $'\r': command not found
: ambiguous redirectine 9: 1
./scripts/runflow: line 11: $'\r': command not found
./scripts/runflow: line 14: syntax error near unexpected token `elif'
'/scripts/runflow: line 14: `elif [ $uses_yarn -eq 0 ]; then
husky > pre-commit hook failed (add --no-verify to bypass)
```

Im using BASH on Windows 10.

**Comment by GregoryHamel** [comment]
@joaquinwojcik, this is a known issue under Windows, see #402. 
For now I would recommend that you do a `yarn flow` and make sure there is no warnings or errors there and then proceed with `yarn cm --no-verify` to do the commit.

**Comment by joaquinwojcik** [comment]
@GregoryHamel PR is ready!

Very very thanks for all the help provided!

**Comment by GregoryHamel** [comment]
@joaquinwojcik, my pleasure! I am not as active on this repo as I used to be, but I am glad to help people keep it alive when I can 👍

**Comment by joaquinwojcik** [comment]
accept attribute is now available! #419
